internal/transform/operation.go

```diff
@@ -1782,6 +1792,24 @@ func contractCodeHashFromTxEnvelope(transactionEnvelope xdr.TransactionV1Envelop
 	return ""
 }
 
+func ledgerKeyHashFromTxEnvelope(transactionEnvelope xdr.TransactionV1Envelope) string {
+	for _, ledgerKey := range transactionEnvelope.Tx.Ext.SorobanData.Resources.Footprint.ReadOnly {
+		ledgerKeyHash := utils.LedgerKeyToLedgerKeyHash(ledgerKey)
+		if ledgerKeyHash != "" {
+			return ledgerKeyHash
+		}
+	}
```

Comment:
we just learned that restore/bump footprint ops will bump all keys in the footprint. Is it possible to store all keys in an array instead of saving only the first one?